js/main.js

```diff
@@ -75,6 +76,28 @@
             supported: !lt10,
             showInputNode: lt10
         });
+
+        // Notifications
+        pulsar.faviconEditor.init();
+
+        // NOTIFICATIONS DEBUG, PLS REMOVE, THX
```

Comment:
To keep in Lexicon or to remove as per comment?

Reply:
I'm hoping to move this over to the Lexicon bundle once that is fixed.